acceptance-tests/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
   testImplementation 'org.apache.logging.log4j:log4j-api'
   testImplementation 'org.assertj:assertj-core'
   testImplementation 'org.awaitility:awaitility'
+  testImplementation 'org.mockito:mockito-core'
```

Comment:
There should be no mocking in Acceptance Tests, why have you added the Mockito core?

Reply:
At some point I needed that, but now I can just remove it and it still works ...

Reply:
Fixed